ddtrace/contrib/pytest_bdd/plugin.py

```diff
@@ -25,9 +25,10 @@ def _store_span(item, span):
     setattr(item, "_datadog_span", span)
 
 
-def pytest_sessionstart(session):
-    if session.config.pluginmanager.hasplugin("pytest-bdd"):
-        session.config.pluginmanager.register(_PytestBddPlugin(), "_datadog-pytest-bdd")
+def pytest_configure(config):
+    # pass
```

Comment:
Delete this?

Reply:
Good catch, I forgot to delete this in testing.